netspresso/clients/auth/client.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+
+import jwt
+import pytz
+from loguru import logger
+
+from netspresso.clients.auth.response_body import TokenResponse, UserResponse
+from netspresso.clients.auth.v1.client import AuthClientV1
+from netspresso.clients.auth.v2.client import AuthClientV2
+from netspresso.clients.config import Config, Module
+
+
+class AuthClient:
+    def __init__(self, config: Config = Config(Module.GENERAL)):
+        """Initialize the UserSession.
+
+        Args:
+            email (str): The email address for a user account.
+            password (str): The password for a user account.
+        """
```

Comment:
p3.
There is not email and paasword in func. params.

Reply:
Thanks for your feedback. I’ve updated the code as per your suggestions. 👍